libs/blocks/merch/autoblock.js

```diff
@@ -55,49 +51,54 @@ export function handleCustomAnalyticsEvent(eventName, element) {
   }
 }
 
-export async function enableAnalytics(el, self) {
+export function cleanupTabsAnalytics(el) {
```

Comment:
the fact that merch autoblock has tabs logic looks weird to me. the analytics logic should be on the tab side IMO.

Reply:
its a hack, yes :(
unfortunately its not so easy to fix tabs analytics since some teams already rely on the way it is working right now.
but for plans page we need a specific behaviour